LabApiUtilities/src/main/com/microsoft/identity/labapi/utilities/client/LabClient.java

```diff
@@ -77,6 +77,13 @@ public ILabAccount getLabAccount(@NonNull final LabQuery labQuery) throws LabApi
         return getLabAccountObject(configInfo);
     }
 
+    public ILabAccount getLabAccount(@NonNull final String upn) throws LabApiException {
+        final List<ConfigInfo> configInfos = fetchConfigsFromLab(upn);
+        // We still get a list of configs despite passing in a single upn, so we select the account from the list.
+        final ConfigInfo configInfo = configInfos.get(0);
+        return getLabAccountObject(configInfo);
```

Comment:
can configInfo be null? seeing if we have to add a null check here

Reply:
I added an assertionError throw in getLabAccountObject. If no lab account is found, we still get a ConfigInfo object, but it has a null userinfo, so this is what I based the assertion on.